app/src/utils/data/grantRounds.ts

```diff
@@ -203,7 +203,15 @@ export async function getGrantRound(blockNumber: number, grantRoundAddress: stri
         matchingToken = SUPPORTED_TOKENS_MAPPING[matchingTokenAddress];
         donationToken = SUPPORTED_TOKENS_MAPPING[donationTokenAddress];
         // record the funds as a human readable number
-        funds = parseFloat(formatUnits(BigNumber.from(funds), SUPPORTED_TOKENS_MAPPING[matchingTokenAddress].decimals));
+        try {
+          funds = parseFloat(
+            formatUnits(BigNumber.from(funds), SUPPORTED_TOKENS_MAPPING[matchingTokenAddress].decimals)
+          );
+        } catch {
+          // If parsing funds for the round fails, ignore the round
+          console.log('Failed to parse funds for round');
+          return {};
```

Comment:
We normally return a grant round which has fields that are expected to be defined based on the type definitions, which aren't necessarily reflected by the LocalForage abstractions—are you sure returning an empty `grantRound` object like this is safe and won't cause other issues?

Reply:
@mds1 what issues do you anticipate? I tested it out and seemed to be fixing the error we're seeing

Reply:
There could also be a more gracious way of fixing the error

Reply:
@mds1 there are a couple of places we use this method like where we update grant rounds. 

We should set a standard way of doing this (if we have not already) throughout our codebase. I.E. if you request an object from a method and it does not exist, fails to retrieve, or has errors, we should consistently return the same thing across the board. That way anywhere we are implementing that method, we can handle the results consistently.